paypro-gateways-woocommerce/includes/paypro/wc/gateway/abstract.php

```diff
@@ -96,7 +96,22 @@ public function process_payment($order_id)
         $product_id = PayPro_WC_Plugin::$settings->productId();
 
         if(empty($paymentDescription))
-            $paymentDescription = 'Order ' . $order->id;
+            $paymentDescription = 'Order ' . $order->get_order_number();
+
+        $test = PayPro_WC_Plugin::$woocommerce->woocommerce3();
+
+        PayPro_WC_Plugin::debug(print_r($test));
```

Comment:
Is this debugging useful for plugin users or is it possibly a leftover from your own debugging?